### ISSUE TITLE
Unable to diff a file with specific diff function

### ISSUE THREAD

**Issue by esseivan** [desktop/desktop#15974]
### The problem

I am unable to diff a zip file (or any with diff=... in .gitattributes).
I set the command in the config (.git/config) file with 
```
[diff "zip"]
	textconv = unzip -v
```
Then in the .gitattributes located in the root of my repository I specified :
`*.zip      diff=zip`

When checking for a diff in Github Desktop, it doesn't load anything and prints an error in the developer tools window
Nothing interesting in the log file.

I tried with other stuff, as `diff=astextplain` but the same thing happens
Also to note that it works with the command line git commands

### Release version

Version 3.1.4 (x64)

### Operating system

Windows 11 x64

### Steps to reproduce the behavior

1. Clean repo with zip file
2. .gitattributes with :
`*.zip      diff=zip`
4. ./.git/config contains the diff :
```
[diff "zip"]
	textconv = unzip -v
```

5. Edit zip file
6. Check diff in Github Desktop for the zip file
7. No diff loaded, infinite loading. Developer tools shows the following error

### Screenshots

![image](https://user-images.githubusercontent.com/14168019/213652751-94715ca1-7012-442c-85ab-75ad22cd96b3.png)

**Comment by esseivan** [comment]
After some debugging I found that the program exits at the line 135 of /app/src/lib/git/spawn.ts.
But I do not get any descriptive message in the console, however, the stderr contained the information I needed : 
`unzip -v: line 1: unzip: command not found\nfatal: unable to read files to diff\n`
So my issue just was that the command I set for the textconv was invalid (even though it works for the cmd-line git). To fix it, I added `C:\Program Files\Git\usr\bin` to the PATH variable.

It could maybe be possible to also print the stderr instead of just printing the default error message (spawn.ts:135) with something like `log.error(stderr.toString());`
